tests/providers/google/common/hooks/test_base_google.py

```diff
@@ -632,6 +643,63 @@ def test_authorize_assert_http_timeout_is_present(self, mock_get_credentials):
         http_authorized = self.instance._authorize().http
         self.assertNotEqual(http_authorized.timeout, None)
 
+    @mock.patch(MODULE_NAME + '.get_credentials_and_project_id')
+    def test_get_credentials_and_project_id_with_impersonation_chain_as_string(
+        self,
+        mock_get_creds_and_proj_id
+    ):
+        mock_credentials = mock.MagicMock()
+        mock_get_creds_and_proj_id.return_value = (mock_credentials, "PROJECT_ID")
```

Comment:
```suggestion
        PROJECT_ID = "PROJECT_ID"
        mock_get_creds_and_proj_id.return_value = (mock_credentials, PROJECT_ID)
```
I would suggest using a global constant for that

Reply:
I was thinking about a global constant, but other tests present in this module were also using hardcoded strings. Should I change them to use the global constant as well? Or just the new tests?

Reply:
New tests will be ok, we can fix others next time